### DISCUSSION TITLE
Would it make sense to show the Wayback machine link only when link is unavailable?

### DISCUSSION THREAD

**Discussion by riipah** [VocaDB/community#164] [Ideas]
Is there a reason to show the wayback machine link even for available links? I'm only thinking of removing clutter, icons and links that are not so necessary. Most the time you'd want to have the wayback machine link if the actual website is not available. Even wayback machine is telling me "This page is available on the web!"

**Comment by ycanardeau** [comment]
Originally I thought it would be useful if there was a link to Wayback Machine so that users can save pages that have not been archived. But yeah, I find it a bit too obtrusive.

**Reply by andreoda** [reply]
I agree
